config/server/nextConfig.ts

```diff
@@ -27,6 +27,7 @@ export function generateNextConfig({directory, dev, port}) {
                 WARZONE_PASSWORD: process.env.WARZONE_PASSWORD,
                 WARZONE_RECAPTCHA_SITE_KEY: process.env.WARZONE_RECAPTCHA_SITE_KEY,
                 WARZONE_RECAPTCHA_SECRET_KEY: process.env.WARZONE_RECAPTCHA_SECRET_KEY,
+                NEXT_PUBLIC_WARZONE_RECAPTCHA_SITE_KEY: process.env.WARZONE_RECAPTCHA_SITE_KEY,
```

Comment:
NEXT_PUBLIC_... makes it so you can access the env var in the front-end

Reply:
Wow that's very convenient